lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -69,6 +70,11 @@ class SPIRVRegularizeLLVMBase {
   // Lower functions
   bool regularize();
 
+  /// Erase cast inst of function and replace with the function.
+  /// Assuming F is a SPIR-V builtin function with op code \param OC.
+  //  void addKernelEntryPoint(Function *F, Op OC);
```

Comment:
Is this comment relevant?

Reply:
oops cut-n-paste fail, wrote a proper constant.